x-pack/plugins/integration_assistant/common/api/generation_error.ts

```diff
@@ -5,6 +5,7 @@
  * 2.0.
  */
 
+import type { ErrorMessageWithLink } from '../../public/components/create_integration/create_integration_assistant/steps/data_stream_step/error_with_link';
```

Comment:
Move this to common instead

Reply:
fixed in https://github.com/elastic/kibana/pull/201792/commits/c01a18cc347a032a738689bc20a40cf075985aed